### DISCUSSION TITLE
大疆DJI无人机等你来拿!蚂蚁集团agentUniverse多智能体框架有奖征文

### DISCUSSION THREAD

**Discussion by LandJerry** [agentuniverse-ai/agentUniverse#168] [Announcements]
![FireShot Capture 022 - 大疆DJI无人机等你来拿!蚂蚁集团agentUniverse多智能体框架有奖征文 - mp weixin qq com](https://github.com/user-attachments/assets/875f9f84-7961-446b-8ebb-5d07df22deae)

原文地址：
https://mp.weixin.qq.com/s/LmG1_1NTkvigdewqE-LQwA

**Comment by kswnia** [comment]
maybe yes,a little bit off the topic,but how i understood this repo with a model only on chinese?

**Reply by LandJerry** [reply]
> maybe yes,a little bit off the topic,but how i understood this repo with a model only on chinese?

Hello, kswnia. The agentUniverse project is aimed at users worldwide. You can use non-Chinese models and technologies, such as OpenAI, Claude, and Ollma, to build intelligent agents.

This topic event is only held in China, which is why Chinese is used. We will have more activities for global developers next year. 

Thank you for your attention. We are looking forward to hearing from you soon.
